Proofreading-OV-Runtime (#11658)
openvinotoolkit/openvino: docs/benchmarks/performance_benchmarks.md

The [Intel® Distribution of OpenVINO™ toolkit](https://software.intel.com/content/www/us/en/develop/tools/openvino-toolkit.html) helps accelerate deep learning inference across a variety of Intel® processors and accelerators.

> The benchmark results below demonstrate high performance gains on several public neural networks on multiple Intel® CPUs, GPUs and VPUs covering a broad performance range. The results can be used to assisit with decision which hardware is best for your applications and solutions or to plan AI workload on the Intel computing already included in your solutions.

Author: ```suggestion
The benchmark results below demonstrate high performance gains on several public neural networks on multiple Intel® CPUs, GPUs and VPUs covering a broad performance range. The results may be helpful when deciding which hardware is best for your applications and solutions or to plan AI workload on the Intel computing already included in your solutions.
```

> Links below consist of the benchmarking results for each alternative to review:

Author: ```suggestion
The links below consist of the benchmarking results for each alternative to review:
```

* [Intel® Distribution of OpenVINO™ toolkit Benchmark Results](performance_benchmarks_openvino.md).
* [OpenVINO™ Model Server Benchmark Results](performance_benchmarks_ovms.md).

> Performance for a particular application can also be evaluated virtually using [Intel® DevCloud for the Edge](https://devcloud.intel.com/edge/). It is a remote development environment with access to Intel® hardware and the latest versions of the Intel® Distribution of the OpenVINO™ Toolkit. You can learn more about it [here](https://www.intel.com/content/www/us/en/developer/tools/devcloud/edge/overview.html) or register [here](https://inteliot.force.com/DevcloudForEdge/s/).

Author: ```suggestion
Performance of a particular application can also be evaluated virtually using [Intel® DevCloud for the Edge](https://devcloud.intel.com/edge/). It is a remote development environment with access to Intel® hardware and the latest versions of the Intel® Distribution of the OpenVINO™ Toolkit. To learn more about it, visit [the website](https://www.intel.com/content/www/us/en/developer/tools/devcloud/edge/overview.html) or [create an account](https://www.intel.com/content/www/us/en/forms/idz/devcloud-registration.html?tgt=https://www.intel.com/content/www/us/en/secure/forms/devcloud-enrollment/account-provisioning.html).
```
